API.md

```diff
@@ -9,18 +9,20 @@
   - [getPendingWaiterState][5]
   - [hasPendingWaiters][6]
   - [\_reset][7]
-- [Test Waiter class][8]
-  - [TestWaiter][9]
+- [Async Test Waiter class][8]
```

Comment:
FWIW, I think we want to avoid mentioning the "class" bit here

Reply:
👍

Reply:
This is actually removed from the docs now. Since we're explicitly documenting publicly exported stuff, this is now gone.